lib/chirp_cockroach/audio/whisper.ex

```diff
@@ -0,0 +1,43 @@
+defmodule ChirpCockroach.Audio.Whisper do
```

Comment:
Only used for caching models/serving - Nx provides a way to define it directly from supervision tree, but it didn't work for me out of the box